lib/render-code.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const prism = require('prismjs');
```

Comment:
The JSON language plugin is not included by default, but often used in our READMEs, so maybe [enable it like so](https://github.com/jbmoelker/jinja-compat-tests/blob/master/lib/nunjucks-filter-prism.js#L2)?:

```javascript
const prism = require('prismjs');
require('prismjs/components/prism-json'); // adds `json` to global `prism.languages` instance
```